client/components/ActivityChart.js

```diff
@@ -1,62 +1,66 @@
-import React from 'react'
+import React from "react";
+import { StyleSheet, View, Dimensions } from "react-native";
 import { connect } from 'react-redux'
-import { Grid, LineChart, XAxis, YAxis } from 'react-native-svg-charts'
-import { View, StyleSheet } from 'react-native'
-import { setActivityWeekThunk } from '../store/activity'
+import { VictoryChart, VictoryArea, VictoryTheme } from "victory-native";
+import Svg, { Text } from 'react-native-svg';
+
+const data = [
```

Comment:
Is data still hard-coded like this? If not, pull it (or at least comment it out) please!